plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -285,38 +350,60 @@ function WordInfoDialog:init()
         buttons = {{reset_button, remove_button}},
         show_parent = self
     }
+
+    local copy_button = Button:new{
+        text = "", -- copy in nerdfont,
+        callback = function() Device.input.setClipboardText(self.title) end,
+        bordersize = 0,
+    }
+    local has_context = self.prev_context or self.next_context
     self[1] = CenterContainer:new{
         dimen = Screen:getSize(),
         MovableContainer:new{
             FrameContainer:new{
                 VerticalGroup:new{
                     align = "center",
                     FrameContainer:new{
```

Comment:
(and it goes on ...) probably the most imbricated one-step widget construction I've seen :) but good if that works and you don't need to reference intermediate widgets (otherwise, you would end up with `self[1][2][1][4]`)